apps/frontend/src/app/backups-overview/backups/backups/backups.component.html

```diff
@@ -36,7 +35,12 @@ <h1 class="header">Backups Overview</h1>
     </clr-datagrid>
   </div>
   <div class="clr-col-sm-12 clr-col-md-6">
-    <div id="backupSizeChart" style="width: 100%; height: 300px"></div>
+    <div class="btn-group">
+      <button class="btn" [class.btn-primary]="selectedTimeRange === 'week'" (click)="setTimeRange('week')">Last Week</button>
+      <button class="btn" [class.btn-primary]="selectedTimeRange === 'month'" (click)="setTimeRange('month')">Last Month</button>
+      <button class="btn" [class.btn-primary]="selectedTimeRange === 'year'" (click)="setTimeRange('year')">Last Year</button>
+    </div>
+    <div id="backupTimelineChart" style="width: 100%; height: 300px"></div>
```

Comment:
y-axis should be in GB
<img width="302" alt="image" src="https://github.com/user-attachments/assets/6878b515-2f23-444d-8e8d-9ed56da73a63">